drivers/shared/executor/executor.go

```diff
@@ -261,9 +261,9 @@ type UniversalExecutor struct {
 // NewExecutor returns an Executor
 func NewExecutor(logger hclog.Logger) Executor {
 	logger = logger.Named("executor")
-	if err := shelpers.Init(); err != nil {
-		logger.Error("unable to initialize stats", "error", err)
-	}
+	// if err := shelpers.Init(); err != nil {
+	// 	logger.Error("unable to initialize stats", "error", err)
+	// }
```

Comment:
Sadly, I am a bit out of my depth here .. there are two calls to `Init()`, but here it seems I cannot access the `cpu_total_compute`. I am not sure how to handle this situation.

I am also not really sure why the `Init` is here; I guess in case it is called before fingerprinting is done; or in cases when no fingerprinting is done at all. But this requires more domain knowledge of Nomad than I have.

Reply:
One thing is sure: without these statements, `raw_exec` job's CPU is not measured :)

Reply:
As I wanted to learn Go a bit more, I delved deeper into this rabbithole. I made a commit that addresses the problem and makes everything work .. but I am not sure it was the right way to go :)

Basically, to get the CpuCompute down here without having access to the configuration, it needs to be send through a complex path. I was hoping for a more elegant solution, but .. here we are :(